mmv1/products/beyondcorp/api.yaml

```diff
@@ -213,3 +213,129 @@ objects:
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
+        description: |
+          ID of the AppConnection.
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'region'
+        description: |
+          The region of the AppConnection.
+        input: true
+        url_param_only: true
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          An arbitrary user-provided name for the AppConnection.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          Resource labels to represent user provided metadata.
+      - !ruby/object:Api::Type::Enum
+        name: 'type'
+        description: |
+          The type of network connectivity used by the AppConnection.
+        values:
+          - :TYPE_UNSPECIFIED
+          - :TCP_PROXY
+        default_value: :TYPE_UNSPECIFIED
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        output: true
+        description: |
+           Represents the different states of a AppConnection.
+        values:
+          - :STATE_UNSPECIFIED
+          - :CREATING
+          - :CREATED
+          - :UPDATING
+          - :DELETING
+          - :DOWN
+      - !ruby/object:Api::Type::NestedObject
+        name: 'applicationEndpoint'
+        description: |
+          Address of the remote application endpoint for the BeyondCorp AppConnection.
+        required: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'host'
+            description: |
+              Hostname or IP address of the remote application endpoint.
+            required: true
+          - !ruby/object:Api::Type::Integer
+            name: 'port'
+            description: |
+              Port of the remote application endpoint.
+            required: true
+      - !ruby/object:Api::Type::Array
+        name: connectors
+        item_type: Api::Type::String
+        description: |
+          List of AppConnectors that are authorised to be associated with this AppConnection
+      - !ruby/object:Api::Type::NestedObject
+        name: gateway
+        description: |
+          Gateway used by the AppConnection.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'appGateway'
+            description: |
+              AppGateway name in following format: projects/{project_id}/locations/{locationId}/appgateways/{gateway_id}.
+            required: true
+          - !ruby/object:Api::Type::Enum
+            name: 'type'
+            description: |
+              The type of hosting used by the gateway.
+            values:
+              - :TYPE_UNSPECIFIED
+              - :GCP_REGIONAL_MIG
```

Comment:
should this value be required ? 
https://cloud.google.com/beyondcorp/docs/reference/rest/v1/projects.locations.appConnections#Gateway

Perhaps not since the tests pass but the documentation might be invalid.